app/helpers/users_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```

Comment:
why this file?

Reply:
I did not add it. It was added by default